bin/yunopaste

```diff
@@ -19,7 +19,7 @@ paste_data() {
   [[ -z "$json" ]] && _die "Unable to post the data to the server."
 
   key=$(echo "$json" \
-    | python -c 'import json,sys;o=json.load(sys.stdin);print o["key"]' \
+    | python -c 'import json,sys;o=json.load(sys.stdin);print(o["key"])' \
```

Comment:
```suggestion
    | python3 -c 'import json,sys;o=json.load(sys.stdin);print(o["key"])' \
```

Let's be explicit and call python3 (just in case for some reason some people have `python-is-python2` installed idk)